fields/field.textbox.php

```diff
@@ -75,9 +75,9 @@ public function isSortable() {
 	-------------------------------------------------------------------------*/
 
 		public function createHandle($value, $entry_id) {
-			$handle = Lang::createHandle(strip_tags(html_entity_decode($value)));
+			$handle = Lang::createHandle(strip_tags(html_entity_decode($value)),160);
```

Comment:
Url are limited to 2083 (technically) Because of IE. https://support.microsoft.com/en-us/kb/208427 so I do not see any problems with 1024.